pkg/scripts/install.go

```diff
@@ -91,10 +92,18 @@ func (i *installer) run(scripts []string, dstDir string) []api.InstallResult {
 		if !i.fs.Exists(file) {
 			sourceResults[script].err = errors.NewDownloadError(file, -1)
 		}
+		file = filepath.Join(dstDir, api.DeprecatedSourceScripts, script)
+		if i.fs.Exists(file) {
+			glog.Infof("DEPRECATED: The .sti/bin directory will be deprecated, use .s2i/bin instead")
+			sourceResults[script] = &downloadResult{location: api.DeprecatedSourceScripts}
+		}
```

Comment:
you should put this first so if they have files in both .s2i and .sti, they get a warning but the files in .s2i get used.